### ISSUE TITLE
Add floating +/- zoom buttons to map

### ISSUE THREAD

**Issue by justinanderson** [CxSci/SA-dashboard#50]
For accessibility, we should have buttons for zooming the map in and out, probably at the bottom right. See [this Figma mockup for an example](https://www.figma.com/file/vP3gxGjxW34i5gNMpu6sDW/Design-V1?node-id=160%3A637).

**Comment by justinanderson** [comment]
Resolved by #53.
